code/game/objects/items/weaponry.dm

```diff
@@ -63,6 +63,8 @@ for further reading, please see: https://github.com/tgstation/tgstation/pull/301
 	flags_1 = CONDUCT_1
 	slot_flags = ITEM_SLOT_BELT | ITEM_SLOT_BACK
 	force = 40
+	damtype = RED_DAMAGE
```

Comment:
All items, by default, are red damage. This is not needed.

Reply:
Specification removed from weapon.